apps/settings/js/bluetooth.js

```diff
@@ -881,7 +881,7 @@ navigator.mozL10n.once(function bluetoothSettings() {
     // enable UI toggle
     gBluetoothCheckBox.disabled = false;
     initialDefaultAdapter();
-    dispatchEvent(new CustomEvent('bluetooth-adapter-added'));
```

Comment:
I think there is no one to listen the event in settings app now. We might remove this line after confirmed nothing broken.